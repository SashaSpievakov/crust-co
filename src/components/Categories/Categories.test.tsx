import '@testing-library/jest-dom';
// import userEvent from '@testing-library/user-event';
// import { screen } from '@testing-library/react';

// import Categories from './Categories';
// import rendererWithStoreAndTheme from '../../tests/helpers/renderer/rendererWithStoreAndTheme';
// import renderWithStoreAndTheme from '../../tests/helpers/render/renderWithStoreAndTheme';

describe('Categories Test', () => {
  // test('renders the Categories component', () => {
  //   const snapshot = rendererWithStoreAndTheme(<Categories />);
  //   expect(snapshot).toMatchSnapshot();
  // });
  // test('renders a correct chosen item', () => {
  //   renderWithStoreAndTheme(<Categories />);
  //   expect(screen.getByRole('listitem', { current: true })).toHaveTextContent(
  //     /all/i,
  //   );
  //   const itemsRest = screen.getAllByRole('listitem', { current: false });
  //   userEvent.click(itemsRest[0]);
  //   expect(screen.getByRole('listitem', { current: true })).toHaveTextContent(
  //     /meat/i,
  //   );
  // });
});
