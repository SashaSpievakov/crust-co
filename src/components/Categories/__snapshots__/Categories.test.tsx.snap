// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Categories Tests renders the Categories component 1`] = `
Object {
  "children": Array [
    <ul
      className="sc-bcXHqe dyLfPm"
    >
      <li
        aria-current={true}
        className="sc-gswNZR kJDdlY"
        onClick={[Function]}
        tabIndex={0}
      >
        All
      </li>
      <li
        aria-current={false}
        className="sc-gswNZR iixEeI"
        onClick={[Function]}
        tabIndex={0}
      >
        Meat
      </li>
      <li
        aria-current={false}
        className="sc-gswNZR iixEeI"
        onClick={[Function]}
        tabIndex={0}
      >
        Vegetarian
      </li>
      <li
        aria-current={false}
        className="sc-gswNZR iixEeI"
        onClick={[Function]}
        tabIndex={0}
      >
        Grill
      </li>
      <li
        aria-current={false}
        className="sc-gswNZR iixEeI"
        onClick={[Function]}
        tabIndex={0}
      >
        Spicy
      </li>
    </ul>,
  ],
  "props": Object {},
  "type": "section",
}
`;
